package.json

```diff
@@ -71,14 +89,15 @@
     "test": "nyc mocha build/test",
     "lint": "gts check && eslint samples/*.js samples/**/*.js",
     "clean": "gts clean",
-    "compile": "tsc -p . && cp src/*.json build/src && cp src/*.js build/src",
+    "compile": "pbjs -t json ./protos/google/longrunning/operations.proto -p ./protos > pbjs-genfiles/operations.json && tsc -p . && cp src/*.json build/src && cp src/*.js build/src ",
```

Comment:
The biggest nit pick of all: there is a space at the end of this string, inside the quotes, maybe it shouldn't be in there

Reply:
OH GOSH NO NO PLEASE NO

(someone please remove this space :) )